### DISCUSSION TITLE
Implementation in Kotlin and Jetpack Compose

### DISCUSSION THREAD

**Discussion by blackbirdsr71br** [mapsforge/mapsforge#1424] [Q&A] [answered]
Good afternoon, I would like to know if this library works in kotlin and with jetpack compose

**Comment by devemux86** [comment, marked as answer]
The [integration guide](https://github.com/mapsforge/mapsforge/blob/master/docs/Integration.md) describes the dependencies.

MapView is an Android View that can be used like all Views with Java / Android compatible languages / frameworks.
